llm/generate/gen_common.sh

```diff
@@ -61,8 +61,8 @@ git_module_setup() {
 
 apply_patches() {
     # Wire up our CMakefile
-    if ! grep ollama ${LLAMACPP_DIR}/examples/server/CMakeLists.txt; then
-        echo 'include (../../../ext_server/CMakeLists.txt) # ollama' >>${LLAMACPP_DIR}/examples/server/CMakeLists.txt
+    if ! grep ollama ${LLAMACPP_DIR}/CMakeLists.txt; then
+        echo 'add_subdirectory(../ext_server ext_server) # ollama' >>${LLAMACPP_DIR}/CMakeLists.txt
```

Comment:
I wonder if this gives us the opportunity to use `FetchContent` now that we have our own target. This would remove the need for a submodule.

Reply:
Wouldn't that require our build to be self-sufficient?  We're riding on all the other discovery logic and compiler/linker setup in the top-level CMakefile.txt.